### DISCUSSION TITLE
Question about Linux build with FFTW included

### DISCUSSION THREAD

**Discussion by Bruce-Yu** [flatironinstitute/finufft#470] [FINUFFT in applications, mathematical definitions]
Hi, I want to build the finufft in Ubuntu with fftw included inside the libnufft.so file. What would be the flags I need to set for compilation? Thanks.

**Comment by lu1and10** [comment]
Hi, you may try to modify the cmakelist or makefile to link againt libnufft.so. But I never try to use fftw with fftw static linked into some other shared lib, is it possible for you to build finufft directly linking fftw?

**Reply by DiamonDinoia** [reply]
You can do `-DFINUFFT_FFTW_LIBRARIES=DOWNLOAD` in cmake and is going to pull fftw build it and bake it into the .so.
